core/src/test/resources/org/apache/calcite/test/RelOptRulesTest.xml

```diff
@@ -5436,49 +5464,42 @@ LogicalProject(EXPR$0=[1])
     </TestCase>
     <TestCase name="testTransitiveInferenceUnion3wayOr">
         <Resource name="sql">
-            <![CDATA[
-select 1 from (
-    select empno, deptno from sales.emp where deptno > 7 or empno < 10
-    union all
-    select empno, deptno from sales.emp where deptno > 10 or empno < deptno
-    union all
-    select empno, deptno from sales.emp where deptno > 1) d
-inner join sales.emp e on d.deptno = e.deptno ]]>
+            <![CDATA[select 1 from (select empno, deptno from sales.emp where deptno > 7 or empno < 10 union all select empno, deptno from sales.emp where deptno > 10 or empno < deptno union all select empno, deptno from sales.emp where deptno > 1) d inner join sales.emp e on d.deptno = e.deptno]]>
```

Comment:
Can you break it into multiple lines? and other SQLs. 
Otherwise, it will be hard to read and maintain.

Reply:
Agree, @yanlin-Lynn , when you update the SQL, please make sure that sql in the test case and xml file are totally in the same format, e.g. they break lines in the same positions.

Reply:
Sure. I add check in `transitiveInference` to make they are totally the same.